jsonrpc/txpool_endpoint.go

```diff
@@ -0,0 +1,153 @@
+package jsonrpc
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/0xPolygon/minimal/types"
+)
+
+// Txpool is the txpool jsonrpc endpoint
+type Txpool struct {
+  d *Dispatcher
+}
+
+type ContentResponse struct {
+  Pending   map[types.Address]map[uint64]*gethTransaction   `json:"pending"`
+  Queued    map[types.Address]map[uint64]*gethTransaction   `json:"queued"`
+}
+
+type InspectResponse struct {
+  Pending   map[string]map[string]string  `json:"pending"`
+  Queued    map[string]map[string]string  `json:"queued"`
+}
+
+type StatusResponse struct {
+  Pending   uint64  `json:"pending"`
+  Queued    uint64  `json:"queued"`
+}
+
+type gethTransaction struct {
+  Nonce       argUint64      `json:"nonce"`
+  GasPrice    argBig         `json:"gasPrice"`
+  Gas         argUint64      `json:"gas"`
+  To          *types.Address `json:"to"`
+  Value       argBig         `json:"value"`
+  Input       argBytes       `json:"input"`
+  V           argByte        `json:"v"`
+  R           argBytes       `json:"r"`
+  S           argBytes       `json:"s"`
```

Comment:
You don't need to put V & R & S in the response according to doc https://geth.ethereum.org/docs/rpc/ns-txpool